src/test/test_confidential_addr.py

```diff
@@ -7,6 +7,47 @@
 CA_PREFIX_LIQUID_REGTEST = 0x04
 EC_PUBLIC_KEY_LEN = 33
 
+hrp = lambda s: s[:s.find('1')]
+
+#   pk=02409b4d18429c6e5cbc0bd59c63b8fe7055f603190c8deed6a644bc95c9772e48
+#   script=OP_HASH160 dc4af3ea14b0592621514e2bd4a0e083c7fac2f2 OP_EQUALVERIFY
+#   -> a914dc4af3ea14b0592621514e2bd4a0e083c7fac2f288
+#   tr hash=409b4d18429c6e5cbc0bd59c63b8fe7055f603190c8deed6a644bc95c9772e48
+segwit_valid_cases = [
```

Comment:
We have some test vectors [here](https://github.com/ElementsProject/elements/blob/master/test/functional/rpc_invalid_address_message.py) if we want to add test coverage also for invalid cases, but it can be done later if you prefer

Reply:
Noted, thanks. I think I will add further liquid tests in another commit unrelated to this one, in parallel with enabling taproot sends on liquid in gdk (this PR has been tested with BTC/gdk to ensure taproot sends work as expected).